File: src/main/java/pemilukm/teti/LoadingWindow.java
package pemilukm.teti;

import javafx.fxml.FXML;
import javafx.scene.Parent;
import javafx.scene.control.Label;
import javafx.scene.layout.VBox;
import javafx.stage.Stage;
import pemilukm.teti.controls.helpers.ControlsHelper;

public class LoadingWindow extends VBox {
    public Stage stage;
    public Parent content;
    @FXML
    private Label mainLabel;

    LoadingWindow() {
        content = ControlsHelper.initControl(this);
    }

    public void setLabelText(String text) {
        mainLabel.setText(text);
    }

}

File: src/main/java/pemilukm/teti/LoadingWindow.kt
package pemilukm.teti

import javafx.fxml.FXML
import javafx.scene.Parent
import javafx.scene.control.Label
import javafx.scene.layout.VBox
import javafx.stage.Stage
import pemilukm.teti.controls.helpers.ControlsHelper

class LoadingWindow internal constructor() : VBox() {
    var stage: Stage? = null
    var content: Parent
    @FXML
    private val mainLabel: Label? = null

    init {
        content = ControlsHelper.initControl(this)
    }

    fun setLabelText(text: String) {
        mainLabel!!.text = text
    }

}
